protocol/contracts/beanstalk/sun/SeasonFacet/Weather.sol

```diff
@@ -12,6 +12,9 @@ import {IBeanstalkWellFunction} from "contracts/interfaces/basin/IBeanstalkWellF
 import {LibWell} from "contracts/libraries/Well/LibWell.sol";
 import {IWell, Call} from "contracts/interfaces/basin/IWell.sol";
 import {IInstantaneousPump} from "contracts/interfaces/basin/pumps/IInstantaneousPump.sol";
```

Comment:
IInstantaneousPump can be removed given `calculateSop` is not used.

Reply:
Removed here: https://github.com/BeanstalkFarms/Beanstalk/commit/7701fc8d1255ef458e82be2282cb6cb0f253f054